lib/modules/platform/gerrit/types.ts

```diff
@@ -30,39 +32,54 @@ export interface GerritBranchInfo {
 
 export type GerritChangeStatus = 'NEW' | 'MERGED' | 'ABANDONED';
 
-export type GerritReviewersType = 'REVIEWER' | 'CC' | 'REMOVED';
+export type GerritRequestDetail =
+  | 'SUBMITTABLE'
+  | 'CHECK'
+  | 'MESSAGES'
+  | 'DETAILED_ACCOUNTS'
+  | 'LABELS'
+  | 'CURRENT_ACTIONS'
+  | 'CURRENT_REVISION'
+  | 'COMMIT_FOOTERS';
 
 export interface GerritChange {
+  _number: number;
   branch: string;
   change_id: string;
   subject: string;
   status: GerritChangeStatus;
   created: string;
-  submittable?: boolean;
-  _number: number;
+  /** Requires o=LABELS. */
+  reviewers?: {
```

Comment:
can you avoid reorder please? I don't see a clear reason for new order.

Reply:
The new order puts the non-optional properties at the top, and the optional properties at the end.

Do you still want me to revert it?

Reply:
we prefer to sort properties alphabetical even if some are optional

Reply:
Got it.

The new order is more alphabetical than before.

Should I restore the older order, sort everything alphabetically, or do nothing?